rpc/chain/client.go

```diff
@@ -350,26 +278,8 @@ func (c *ClientWithFallback) BlockNumber(ctx context.Context) (uint64, error) {
 	rpcstats.CountCallWithTag("eth_BlockNumber", c.tag)
 
 	res, err := c.makeCall(
-		ctx, c.ethClients, func(client *EthClient) (interface{}, error) {
-			return client.ethClient.BlockNumber(ctx)
-		},
-	)
-
-	c.toggleConnectionState(err)
```

Comment:
Was this removed intentionally? I don't see call to `c.toggleConnectionState` in the new code.

Reply:
I think it's just confusing git diff UI, because some methods have been removed:

`toggleConnectionState` in BlockNumber is here:

https://github.com/status-im/status-go/blob/2f71d9d9f2529cfa21cabb216be7992a5f9f6717/rpc/chain/client.go#L358

Reply:
yeap, it's hidden but it's there